assets/api/writer-generator/python/requirements.txt

```diff
@@ -1,5 +1,6 @@
 requests>=2.32.0,<3.0.0
 pytest>=8.3.0,<9.0.0
 pydantic>=2.11.0,<3.0.0
+fhirpy>=2.2.0
```

Comment:
Generated types shouldn't be dependent on the fhirpy package, and as I understand it -- we don't actually need it. It should be a part of the example.